partcad/src/partcad/wrappers/wrapper_openscad.py

```
@@ -0,0 +1,80 @@
+        lines = f.readlines()
+
+    base_dir = os.path.dirname(path)
+    for line in lines:
+        line_stripped = line.strip()
+        if line_stripped.startswith("use <") or line_stripped.startswith("include <"):
+
+            file_name = line_stripped.split("<")[1].split(">")[0]
+            file_path = os.path.join(base_dir, file_name)
+
+            if os.path.exists(file_path):
+                shutil.copy(file_path, "/tmp")
```

Ruff (0.8.2)

Probable insecure usage of temporary file or directory: "/tmp"

(S108)